### PR TITLE
[#2393] Update the Wizzard Access error msg

### LINKED ISSUE #2393: More descriptive error when adding open access to the project second time
According to the discussion in the #2066 . 

The logic itself (user can not add open access resource without technical parameters more than once to the same project) is correct. But the error message should be more descriptive - **"Already pinned in the project <xyz>"**

image

### PR DESCRIPTION
**Pull request by danielkryska**
Closes #2393

### CODE REVIEW

**Comment by danielkryska** [comment]
localhost_5000_services_alien-and-invasive-species-virtual-research-environment_summary

File: config/locales/en.yml [hunk 1]
```diff
@@ -113,4 +113,4 @@ en:
         project_item/wizard/summary_step:
           attributes:
             project:
-              repeated_in_project: "can only have one of this resource. Please choose another project"
+              repeated_in_project: "already pinned to the service %{service_name}"
```
Comment (kmarszalek) [inline config/locales/en.yml:116]: ``
repeated_in_project:
format: "Already pinned to the project"
``
and add config.active_model.i18n_customize_full_message = true in config/application.rb and you can make your own custom msgs :) 

source: https://www.bigbinary.com/blog/rails-6-allows-to-override-the-activemodel-errors-full_message-format-at-the-model-level-and-at-the-attribute-level

**Review by kmarszalek — DISMISSED** [review]
I would make a custom msg, it works now. not sure why it wasn't working previously.

**Comment by kmarszalek** [comment]
rebase and changelog :)

File: config/locales/en.yml (hunk 1 above)
Comment (jswk) [reply config/locales/en.yml:116]: Rather:
```
project:
  repeated_in_project: "Already pinned to the project"
  format: "%{message}"
```

**Review by jswk — DISMISSED** [review]
> rebase and changelog :)

Rebasing should also remove the superfluous commit.

File: config/locales/en.yml (hunk 1 above)
Comment (kmarszalek) [reply config/locales/en.yml:116]: also works 🥇

**Comment by kmarszalek** [comment]
> > rebase and changelog :)
> 
> Rebasing should also remove the superfluous commit.

that's the reason! :)

File: config/locales/en.yml (hunk 1 above)
Comment (danielkryska) [reply config/locales/en.yml:116]: Sometimes I'm wondering if you're kind of evangelist... Have you considered that:
- format on the property level doesn't work
- forcing message template on the `project` model level will overwrite all error messages?
- the ticket is focusing on the simple change of text and creating additional layers of abstract magic is overthinking
- after a quick talk with the @michal-szostak value of the error content MUST NOT BE EXACTLY SAME AS IN TICKET

Comment (danielkryska) [reply config/locales/en.yml:116]: SUMMARIZING -> DO NOT OVERTHINK !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

Comment (kmarszalek) [reply config/locales/en.yml:113]: caps lock and exclamation marks aren't proper forms of conversation.

> * format on the property level doesn't work

well then pray tell, how I was able to make it work?

> * forcing message template on the `project` model level will overwrite all error messages?

on project model, yes. ok but.. you have all two messages for the project...

> * the ticket is focusing on the simple change of text and creating additional layers of abstract magic is overthinking

These additional layers of abstract magic make it work as intended in the ticket.

> * after a quick talk with the @michal-szostak value of the error content MUST NOT BE EXACTLY SAME AS IN TICKET

and where can I find any mention of that, as a reviewer I am looking in the ticket and PR description. 
if you do not put any effort into documenting your conversation or conclusions, how do you expect anyone to take it into account?

I am not an evangelist, I tried to make custom msg work a month ago but some dependency was able to thwart my effort. now it works, and it's not even too much work.

Zrzut ekranu 2021-11-17 o 09 28 27
Zrzut ekranu 2021-11-17 o 09 28 38

**Comment by danielkryska** [comment]
The final error message version example:
`Project already pinned with the offer: General access`

**Comment by Marcelinna** [comment]
the case when we have default offer
image

**Comment by Marcelinna** [comment]
Taking into account the problems with the name of the offers, let's change to the **Project already pinned with this offer**

**Review by Marcelinna — APPROVED** [review]

**Review by michal-szostak — APPROVED** [review]

**Pull request merged**